### PR TITLE
add versionStage for secretdata

### PR DESCRIPTION
**Pull request by Yidingluo1**
versionstage is used to mark the status of versions for the secretdata.
by default, the latest version is marked as current(1), and the others are previous(0).

test: passed the unittest

Signed-off-by: yidingluo1 <luo.yd@neusoft.com>

### CODE REVIEW

**Comment by syan10** [comment]
the commit msg is not make sense.
you need to explain more details about the changes, such as current/previous meaning?

**Comment by Yidingluo1** [comment]
> the commit msg is not make sense. you need to explain more details about the changes, such as current/previous meaning?

Commit changed

**Review by syan10 — COMMENTED** [review]
change the title as 
"
add versionStage for secretdata


versionstage is used to mark the status of versions for the secretdata.
by default, the latest version is marked as current(1), and the others are previous(0).

test: passed the unittest
"

**Comment by Yidingluo1** [comment]
> change the title as " add versionStage for secretdata
> 
> versionstage is used to mark the status of versions for the secretdata. by default, the latest version is marked as current(1), and the others are previous(0).
> 
> test: passed the unittest "

Changed as required

**Review by syan10 — APPROVED** [review]

File: ehsm_kms_service/router.js [hunk 1]
```diff
@@ -252,12 +252,12 @@ const router = async (p) => {
...
       } catch (error) {}
       break
-    default:
-      res.send(_result(404, 'Not Found', {}))
-      break
     case secret_manager_apis.CreateSecret:
         createSecret(res, appid, payload, DB)
       break
+    default:
+      res.send(_result(404, 'Not Found', {}))
```
Comment (HawieLong) [inline ehsm_kms_service/router.js:259]: API Not Found

Comment (Yidingluo1) [reply ehsm_kms_service/router.js:259]: it will be changed

**Review by yang8621 — APPROVED** [review]

**Pull request merged**